dojo/importers/default_importer.py

```diff
@@ -0,0 +1,472 @@
+            old_findings = old_findings.filter(Q(service__isnull=True) | Q(service__exact=''))
+        # Determine if pushing to jira or if the finding groups are enabled
+        push_to_jira = kwargs.get("push_to_jira", False)
+        finding_groups_enabled = is_finding_groups_enabled()
+        # Update the status of the findings and any endpoints
+        for old_finding in old_findings:
+            self.mitigate_finding(
+                old_finding,
+                user,
+                scan_date,
+                (
+                    "This finding has been automatically closed "
+                    "as it is not present anymore in recent scans."
+                ),
+                finding_groups_enabled,
+                push_to_jira,
+            )
+        # push finding groups to jira since we only only want to push whole groups
+        if finding_groups_enabled and push_to_jira:
+            for finding_group in {finding.finding_group for finding in old_findings if finding.finding_group is not None}:
+                jira_helper.push_to_jira(finding_group)
+
+        return old_findings
+
+    def parse_findings_static_test_type(
+        self,
+        parser: Parser,
+        scan_type: str,
+        scan: TemporaryUploadedFile,
+        test: Test = None,
+        **kwargs: dict,
+    ) -> Tuple[Test, List[Finding]]:
+        """
+        Creates a test object as part of the import process as there is not one present
+        at the time of import. Once the test is created, proceed with the traditional
+        file import as usual from the base class
+        """
+        # by default test_type == scan_type
+        test = self.create_test(
+            scan_type,
+            scan_type,
+            **kwargs,
+        )
+        logger.debug('IMPORT_SCAN: Parse findings')
+        # Use the parent method for the rest of this
+        return test, BaseImporter.parse_findings_static_test_type(
+            self,
+            parser,
+            scan_type,
+            scan,
+            test=test,
+            **kwargs,
+        )
+
+    def parse_findings_dynamic_test_type(
+        self,
+        parser: Parser,
+        scan_type: str,
+        scan: TemporaryUploadedFile,
+        **kwargs: dict,
+    ) -> Tuple[Test, List[Finding]]:
+        """
+        Uses the parser to fetch any tests that may have been created
+        by the API based parser, aggregates all findings from each test
+        into a single test, and then renames the test is applicable
+        """
+        logger.debug('IMPORT_SCAN parser v2: Create Test and parse findings')
+        parsed_findings = []
+        tests = self.parse_dynamic_test_type_tests(
+            parser,
+            scan_type,
+            scan,
+            **kwargs,
+        )
+        # Make sure we have at least one test returned
+        if len(tests) == 0:
+            logger.info(f'No tests found in import for {scan_type}')
+            return parsed_findings
```

Comment:
Not sure if `None` is the right thing to return here, but to ensure 2 values are returned in this case

```suggestion
            return None, parsed_findings
```

Reply:
Good find! Addressing on the next commit